src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/TimePicker/Sample1.xaml

```diff
@@ -11,28 +11,21 @@
       xmlns:controls="using:Uno.UI.Samples.Controls"
       mc:Ignorable="xamarin android ios native_ios native_android">
 
-    <controls:SampleControl SampleDescription="Sample1">
-        <controls:SampleControl.SampleContent>
-            <DataTemplate>
-                <Grid Background="White">
-                    <Grid.RowDefinitions>
-                        <RowDefinition Height="Auto" />
-                        <RowDefinition Height="*" />
-                    </Grid.RowDefinitions>
-                    	<Button Content="Set To Current Time"
-							Command="{Binding SetToCurrentTime}" Margin="10" HorizontalAlignment="Center" Visibility="Collapsed"/>
-                    <TimePicker Grid.Row="1"
+	<controls:SampleControl SampleDescription="style:default - clock:24h - increment:1 - time:now">
+		<controls:SampleControl.SampleContent>
+			<DataTemplate>
+				<Grid Background="White">
+					<TimePicker Grid.Row="1"
                                 Margin="15,10"
                                 ios:FlyoutPlacement="Full"
                                 HorizontalAlignment="Center"
                                 VerticalAlignment="Top"
-                                Time="{Binding Time, Mode=TwoWay}"
-                                MinuteIncrement="1"
+                                MinuteIncrement="13"
```

Comment:
Shouldn't Sample1 have a minute increment of 1 or just no specifications at all ?  (be more like default)

Reply:
SampleDescription says 1